docs/_examples/evaluation/simulate/plots.py

```diff
@@ -0,0 +1,120 @@
+from pathlib import Path
+import sys
+
+from matplotlib import pyplot as plt
+
+import numpy as np
+
+from sherpa.utils.random import poisson_noise
+
+# Assume run from docs directory.
+#
+indir = Path("../sherpa-test-data/sherpatest")
+outdir = Path("_static/evaluation/")
+
+for wanted in [indir, outdir]:
+    if not wanted.is_dir():
+        sys.stderr.write(f"ERROR: no directory {wanted}\n")
+        sys.exit(1)
+
+def savefig(name):
+    plt.savefig(f"_static/evaluation/{name}.png")
+    print(f"** Created: {name}.png")
+
+data_dir = str(indir)
+if not data_dir.endswith("/"):
+    data_dir += "/"
+
```

Comment:
Instead of this, you could also do the following below:
`os.path.join(data_dir, '9774_bg.pi')`
That will take care of the "/" at the end of the data_dir string or not. It will also work on windows, which uses "\" as a separator. (Not that we currently support windows, but just saying in general...)

Reply:
As I mentioned, I think the last commit is going to get removed for 4.17.0, but useful to think about for the future.

Reply:
Also, this is meant to replicate the code in `docs/evaluation/simulate.rst` and so I only want to change the "setup" code (since `data_dir` in the actual documentation is provided out-of-band).